selfdrive/car/hyundai/values.py

```diff
@@ -33,7 +33,7 @@ class CAR:
   SONATA = "HYUNDAI SONATA 2020"
   SONATA_LF = "HYUNDAI SONATA 2019"
   PALISADE = "HYUNDAI PALISADE 2020"
-  VELOSTER = "HYUNDAI VELOSTER 2019"
+  VELOSTER = "HYUNDAI VELOSTER 2019-2020"
```

Comment:
Revert this change, only update the model year in the readme

Reply:
Done!